CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+#### 0.10.0 (2024-05-27)
+
+* Revert `WithSetup` variants that were introduced in 0.7.2. Due to the way
+  repetitions are done now, there is no need for these variants anymore (PR #43);
+
+* Delete `run[N]ParallelCommandsNTimes` in favor of the new
+  `forAll[N]ParallelCommandsNTimes` as repetitions now happen at the `forAll`
+  level (PR #43);
+
+* Rename `forall` to `forAll` (PR #49).
+
+* Replace `ansi-wl-pprint` with `prettyprinter` (PR #49).
```

Comment:
Same here, explain why: i.e. because ansi-wl-pprint is deprecated in favour of prettyprinter.

Reply:
Done